tests/testlogout.js

```diff
@@ -0,0 +1,31 @@
+import LoginPage from '../pages/login.page';
+import HomePage from '../pages/home.page';
+import LogoutPage from '../pages/logout.page';
+//import { expect } from 'chai';
+
+describe ('TestLogout', async () => {
+
+    it('Should login and logout successfully', async () => {
+
+        await LoginPage.open('login');
+        await LoginPage.loginAs('jsmith@demo.io', 'Demo123!');
+        await expect(HomePage.welcomeUser).toHaveTextContaining('Welcome Josh');
+        
+        await LogoutPage.logout();
+        
+
+    });
+
+    it('Should login successfully and logout pressing the back arrow', async () => {
```

Comment:
When you press the "back" button you are not really logging out, only returning to the login page, so consider removing this test, as it's only asserting that you can return to the previous page, but logging out is not happening.